internal/serialization/serialization_consts.go

```diff
@@ -100,155 +100,85 @@ const (
 	TypeNotDecoded = -2024
 )
 
+var typeToString = map[int32]string{
+	TypeNil:                                  "NIL",
+	TypePortable:                             "PORTABLE",
+	TypeDataSerializable:                     "DATA_SERIALIZABLE",
+	TypeByte:                                 "BYTE",
+	TypeBool:                                 "BOOL",
+	TypeUInt16:                               "UINT16",
+	TypeInt16:                                "INT16",
+	TypeInt32:                                "INT32",
+	TypeInt64:                                "INT64",
+	TypeFloat32:                              "FLOAT32",
+	TypeFloat64:                              "FLOAT64",
+	TypeString:                               "STRING",
+	TypeByteArray:                            "BYTE_ARRAY",
+	TypeBoolArray:                            "BOOL_ARRAY",
+	TypeUInt16Array:                          "UINT16_ARRAY",
+	TypeInt16Array:                           "UINT16_ARRAY",
```

Comment:
It could be "INT16_ARRAY"?

Reply:
Thanks, fixed at: https://github.com/hazelcast/hazelcast-commandline-client/pull/159/commits/ff8e67dd3673dbb32a0005c0509f3b81dc9944a4